components/constants/webui_url_constants.h

```diff
@@ -57,5 +57,7 @@ extern const char kPlaylistHost[];
 extern const char kPlaylistURL[];
 extern const char kSpeedreaderPanelURL[];
 extern const char kSpeedreaderPanelHost[];
+extern const char kTooltipURL[];
+extern const char kTooltipHost[];
```

Comment:
kTooltipHost looks kinda confusing to me as we have Tooltips for views. Could kPromoUI or kUserEducation be better?

Reply:
changing it to `kFeatureAnnouncementTooltipURL` and `kFeatureAnnouncementTooltipHost`